_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -107,9 +107,22 @@ FIXME:harding to update
 [Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
 (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
-- [Bitcoin Core #20564][] and [#20612][Bitcoin Core #20612] Don't send
-  'sendaddrv2' to pre-70016 software, and send before 'verack'
-  FIXME:jnewbery, maybe also mention mailing list post and #20599
+- [Bitcoin Core #20564][] makes two changes to the way that Bitcoin Core
+  signals support for `addrv2` messages ([BIP155][]):
+
+  - Bitcoin Core will only negotiate support for `addrv2` messages with peers that signal
+    they are using p2p version 70016 or higher. This restriction isn't required by the BIP,
+    but release testing has revealed that some other implementation will disconnect
+    from Bitcoin Core if they receive any unknown message, including
+    `sendaddrv2`. This change may be reverted in future versions of Bitcoin Core,
+    so other implementations are advised to always tolerate unknown p2p messages.
+
+  - The `sendaddrv2` message will now be sent between the `version` and
+    `verack` message, as required by the latest version of BIP155. See [BIPs
+    #1043](#bips-1043) below for more information about that change to the BIP.
+
+    This PR was back-ported to [the latest V0.21 release candidate][Bitcoin
```

Comment:
s/back-ported/backported/? (e.g. do we write back-packs vs backpacks, but maybe it's English English)